app/assess/tag_routes.py

```diff
@@ -263,3 +265,47 @@ def reactivate_tag(fund_id, round_id, tag_id):
         tag_config=Config.TAGGING_PURPOSE_CONFIG,
         fund_round=fund_round,
     )
+
+
+@tag_bp.route(
+    "/tags/edit/<fund_id>/<round_id>/<tag_id>", methods=["GET", "POST"]
+)
+@check_access_fund_id(roles_required=["ASSESSOR"])
+def edit_tag(fund_id, round_id, tag_id):
+    edit_tag_form = EditTagForm()
+    fund_round = get_fund_round(fund_id, round_id)
+    tag = get_tag(fund_id, round_id, tag_id)
+    if request.method == "GET":
+        current_app.logger.info(f"Loading edit tag page for id {tag_id}")
+
+    elif request.method == "POST":
+        current_app.logger.info("In edit tag put")
+        if edit_tag_form.validate_on_submit():
+            # Save changes
+            payload = [{"id": tag_id, "value": edit_tag_form.value.data}]
+            result = update_tag(fund_id, round_id, payload)
```

Comment:
If you follow my other suggestion, you could remove the list nesting here:

```suggestion
            payload = {"id": tag_id, "value": edit_tag_form.value.data}
            result = update_tag(fund_id, round_id, payload)
```